components/DropdownLight/DropdownLight.jsx

```diff
@@ -10,9 +12,76 @@ const itemPropType = PropTypes.oneOfType([
   }),
 ]);
 
-const DropdownSelect = styled.select`
-  width: 100%;
-  height: 44px;
+const ITEM_HEIGHT = '30px';
+const MAX_ITEMS_VISIBILITY = 7;
+
+const CheckIcon = styled(Icon).attrs({
+  name: 'check',
+})`
+  ${({
+    selectedItem,
+    theme: {
+      colors: {
+        primary: { 700: primary700 },
+      },
+    },
+  }) =>
+    !selectedItem &&
+    `
+    color: ${primary700};
+  `}
+`;
+
+const ArrowIcon = styled(Icon)`
+  display: inline-block;
+  pointer-events: none;
+  width: 24px;
+`;
+
+const ButtonField = styled.button`
+  width: ${components.container.breakpoints.small};
+  padding: ${spacing.xsmall}px ${spacing.medium}px;
+  border-radius: ${spacing.xxsmall}px;
```

Comment:
I believe it is not necessary to put the spacing in this case

Reply:
it's necessary

Reply:
Spacing is only for spacing props, like `margin` and `paddings`.

Reply:
I understand, thanks guys